src/webhooks/registry.ts

```diff
@@ -107,11 +142,14 @@ function buildQuery(
   let mutationName: string;
   let webhookSubscriptionArgs: string;
   switch (deliveryMethod) {
-    case DeliveryMethod.Http:
+    case DeliveryMethod.Http || !versionSupportsEndpointField():
       mutationName = webhookId ? 'webhookSubscriptionUpdate' : 'webhookSubscriptionCreate';
       webhookSubscriptionArgs = `{callbackUrl: "${address}"}`;
       break;
     case DeliveryMethod.EventBridge:
+      if (!versionSupportsEndpointField()) {
+        throw new ShopifyErrors.UnsupportedClientType();
```

Comment:
Nit: let's add an error message here to help make the reason for the error clearer.

Reply:
I've added `EventBridge webhooks are not supported in API version "${Context.API_VERSION}".` as the message, combined with the change suggested by @mkevinosullivan.